docs/tutorials/aws/AIConnectorHelper.ipynb

```diff
@@ -0,0 +1,1069 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5f2363c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Are these lines really needed for { "cells": ...} ? They seems not appearing in your own repo?

Reply:
Yes, they need these